src/reducers/auth.reducer.js

```diff
@@ -0,0 +1,28 @@
+import isEmpty from 'lodash/isEmpty';
+import { SET_CURRENT_USER, LOG_OUT_USER } from "../actions/action.types";
+
+const initialState = {
+  isAuthenticated: true,
```

Comment:
Hello, @CEOehis Nice job so far. Please, I'd like to know why isAuthenticated is set to true at the initial State. I thought it should be false

Reply:
Thanks for pointing this out. I'll fix this ASAP.